CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.1.6
```

Comment:
> Although semantic versioning doesn’t promise any compatibility between versions prior to 1.0.0, the Dart community convention is to treat those versions semantically as well. The interpretation of each number is just shifted down one slot: going from 0.1.2 to 0.2.0 indicates a breaking change, going to 0.1.3 indicates a new feature, and going to 0.1.2+1 indicates a change that doesn’t affect the public API. For simplicity’s sake, avoid using + after the version reaches 1.0.0.

https://dart.dev/tools/pub/versioning#semantic-versions

Let's make all breaking changes increase the minor version number (as long as we're not yet at 1.0).

Reply:
Yeah